Modules/_datetimemodule.c

```diff
@@ -1873,6 +1878,11 @@ wrap_strftime(PyObject *object, PyObject *format, PyObject *timetuple,
     pnew = PyBytes_AsString(newfmt);
     usednew = 0;
 
+    PyObject *strftime = _PyImport_GetModuleAttrString("time", "strftime");
+
+    if (strftime == NULL)
+        goto Done;
```

Comment:
Style nitpick:
```suggestion
    if (strftime == NULL) {
        goto Done;
    }
```

Reply:
I understand the argument for this style even though there are a good number of unbracketed single-line `if` bodies right above this code. But you're right that it's good habit to keep for sure.